x-pack/plugins/cloud_security_posture/public/test/utils.ts

```diff
@@ -5,13 +5,21 @@
  * 2.0.
  */
 
-import { screen } from '@testing-library/react';
+import { Matcher, screen } from '@testing-library/react';
+
+export const expectIdsInDoc = ({
+  be = [],
+  notToBe = [],
+}: {
+  be: Matcher[];
+  notToBe?: Matcher[];
+}) => {
+  const filteredNotToBe = notToBe.filter((testId) => !be.includes(testId));
```

Comment:
this change allows this function to be used a bit more easily, instead of providing individual ids, you can just pass an entire array and select which id from it you want to exist in the doc. it does not effect the previous way we used it so it's just an enchantment.